### PR TITLE
Support cuTENSOR 1.5.0

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

File: cupyx/tools/install_library.py [hunk 1]
```diff
@@ -124,38 +124,21 @@ def _make_cutensor_record(
...
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.0', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '10.2', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
+def _make_cutensor_record(cuda_version):
```
Comment (takagi) [inline cupyx/tools/install_library.py:127]: cuTENSOR provides the same headers and binaries for each CTK version it supports these days, so I wrapped up the calls to make cutensor records.

**Comment by leofang** [comment]
The cuda110 error should be fixed now (https://github.com/cupy/cupy/pull/6671). Let's retry...

/test full

**Comment by leofang** [comment]
CI is green 👍

**Comment by takagi** [comment]
@emcastillo I will make this backport.

**Pull request merged**